.github/workflows/main.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Toda vez que der push na main vai rodar esse código? E se for um merge na main por um PR funciona tb?